examples/vlan_learning.c

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) PLUMgrid, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License")
+
+#include <bcc/proto.h>
+
+struct ifindex_leaf_t {
+  int out_ifindex;
```

Comment:
there is a 4 byte gap in the struct. I guess llvm decided that it's cheaper to initialize the whole thing including gaps. Better to avoid.